drivers/led_strip/Kconfig.ws2812

```diff
@@ -39,6 +38,7 @@ config WS2812_STRIP_GPIO
 	# Only an Cortex-M0 inline assembly implementation for the nRF51
 	# is supported currently.
 	depends on SOC_SERIES_NRF51X
+	select LED_STRIP_RGB_SCRATCH
```

Comment:
The scratch byte is only used in the ws2812_gpio driver due to a programming error. The WS2812 LED strip device does not need a scratch byte. Please remove it for all the WS2812 drivers.

Reply:
So doesn't this mean the whole Kconfig is wrong and the only driver still using it apa102, is wrong too?

Reply:
Actually, you are right. I forgot about the white byte thing :( Four bytes are actually sent over the wire when there is a white byte (RGBW mapping for example). The scratch byte is needed to cover this fourth byte. And that somehow matches the description in Kconfig...

Please rollback your patch to its previous version. It was actually correct.

I apologize for the inconvenience.

Reply:
Reverted